airbyte-integrations/connectors/source-typeform/source_typeform/source.py

```diff
@@ -83,8 +83,13 @@ def request_params(
 
 class TrimFormsMixin:
     def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
-        for item in TrimForms(**self.config).read_records(sync_mode=SyncMode.full_refresh):
-            yield {"form_id": item["id"]}
+        form_ids = self.config.get("form_ids", [])
```

Comment:
check connection should verify the forms input by the user are valid and if not point out which forms are invalid

Reply:
+1